source/isaaclab/isaaclab/sensors/frame_transformer/frame_transformer.py

```diff
@@ -416,45 +423,82 @@ def _update_buffers_impl(self, env_ids: Sequence[int]):
         self._data.target_pos_source[:] = target_pos_source.view(-1, total_num_frames, 3)
         self._data.target_quat_source[:] = target_quat_source.view(-1, total_num_frames, 4)
 
+    def _draw_lines(self, source_pos, target_pos):
+        # Calculate the direction vector and length
```

Comment:
```suggestion
    def _draw_lines(self, source_pos: torch.Tensor, target_pos: torch.Tensor):
        """Draws lines connecting source and target transforms."""
        # Calculate the direction vector and length
```

Reply:
We typically put helper functions towards the end of the file so that the reading order when going through the files is: public operations, overridden internal functions, internal helper functions.

Could you please move this down? :)

Reply:
@Mayankm96 Thanks for pointing that out. I have moved them down.